www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/httpserver.xml

```diff
@@ -132,6 +132,12 @@
     <type>checkbox</type>
     <help>If you check this box, a TLS encrypted connection is enforced.</help>
   </field>
+  <field>
+    <id>httpserver.ocsp_stapling</id>
+    <label>OCSP Stapling</label>
+    <type>checkbox</type>
+    <help>If you check this box, OCSP stapling (ssl_stapling) is enabled.</help>
```

Comment:
```suggestion
    <help>If you check this box, OCSP stapling is enabled.</help>
```

I never mention the real configuration directives and the nginx plugin frontend is entirely SSL free since it is an outdated and disabled protocol in the plugin.

Reply:
What do you think about mentioning the problem in the linked ticket (browser rejects the connection as it does not get an OCSP response).

This would be far more helpful in the help text.

And yes, please mark this setting as advanced.